### DISCUSSION TITLE
Sharing My Experience with react-virtuoso

### DISCUSSION THREAD

**Discussion by taekyeomlee** [petyosi/react-virtuoso#1063] [General]
First of all, thank you for creating such an excellent library.

I recently applied list virtualization to my project using react-virtuoso. My personal experience was that the library's completeness is outstanding. However, due to the lack of references, I spent a lot of time searching for materials and checking the documentation. **Therefore, I'd like to share my experience and examples. I'm also curious if it's possible to reflect this example on GitHub or in the official documentation.**

The situation is as follows:

- A list page (infinite scroll + list virtualization)
- A detail page

This is about maintaining the scroll position when selecting an item on the list page, visiting its detail page, and then returning.

I'm planning to use TanstackQuery for the infinite scroll and React Router Dom for routing.

Since this is my first time attempting such a contribution, **I would appreciate any guidance.**

**Comment by petyosi** [comment]
Hi @taekyeomlee it's great that you want to give back and help with what you've learned. Fortunately, it's also quite easy. 

- Create a working example of what you've done in https://codesandbox.io/. Ensure that the example does not overcomplicate things by removing non-essential details. 
- Update your post to include the link and any necessary details that can be beneficial for the users to pick it up.
- Change the category to show and tell - I can do that too if GitHub does not allow it :).

That should be it! 🙌🏼 . Thank you very much for your help.
